integration_test.go

```diff
@@ -60,6 +60,12 @@ func (srv *serverlessAgent) HandleBundle(w http.ResponseWriter, req *http.Reques
 		body = nil
 	}
 
+	if value, ok := os.LookupEnv("NEED_ERROR"); ok && value == "true" {
```

Comment:
Query: Is it possible to remove env wide setting?

Reply:
changed the settings into a tag in spans.